README.md

```diff
@@ -1,32 +1,116 @@
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+<h1 align="center" >:vhs:</h1> 
+<h3 align="center">VHS</h3>
+
+  <p align="center">
+    The cloud-native swiss army knife for network traffic data.
+    <br />
+    <a href="https://rename-this.github.io/vhs/"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/rename-this/vhs/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rename-this/vhs/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary><h2 style="display: inline-block">Table of Contents</h2></summary>
+  <ol>
+    <li><a href="#about-the-project">About The Project</a></li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#demo-setup">Demo Setup</a></li>
+        <li><a href="#run-the-demo">Run The Demo</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#built-with">Built With</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+`vhs` is a versatile tool for network traffic capture. `vhs` can be run as a command line tool or be deployed into your
+Kubernetes cluster as a sidecar, where it can capture traffic to and from your services. Captured traffic
+can be analyzed to produce live Prometheus metrics or saved for use in offline analysis, load testing, or whatever
+you can imagine!
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+The quickest way to see `vhs` in action is to use the development Docker image in this repo to run a demo.
+
+### Prerequisites
+The first step is to clone the VHS repository.
 
-Record and replay HTTP traffic
-
-# Development
+```
+git clone https://github.com/rename-this/vhs.git
+```
 
-The easiest way to run this is to use the docker image included in the repo.
+You will need a working Docker installation to successfully execute the following commands. You should be able to
+install Docker from your favorite package manager, or you can see the [Docker site](https://www.docker.com/get-started)
+for more details.
 
-To build the image (which installs some helpful tools and mounts the source in the container):
+Once you have Docker set up and working, you can build the development container (which installs some helpful tools
+and mounts the source in the container) by changing directory into the repository you cloned and running:
 
 ```
 $ make dev
 ```
 
-Open two bash sessions on the container. 
+### Demo Setup
+Open a bash session on the container by running the following command in a terminal:
 
-In one, run this script to boot a simple echo server and a curl request that calls the server every second:
+```
+$ docker exect -it vhs_dev bash
```

Comment:
```suggestion
$ docker exec -it vhs_dev bash
```

Reply:
Based on other comments above, the demo section of the README will now just link to the getting started page on the docs site, so this section will go away.